cfme/tests/containers/test_containers_data_integrity_properties.py

```diff
@@ -0,0 +1,114 @@
+import pytest
+from cfme.fixtures import pytest_selenium as sel
+from itertools import product
+from cfme.web_ui import CheckboxTable
+from cfme.containers.image import Image
+from cfme.containers.image_registry import ImageRegistry
+from cfme.containers.pod import Pod
+from cfme.containers.provider import Provider
+from cfme.containers.node import Node
+from cfme.containers.container import Container
+from utils import testgen
+from utils.version import current_version
+
+list_tbl = CheckboxTable(table_locator="//div[@id='list_grid']//table")
```

Comment:
The appropriate list_tbl objects are defined in `cfme.containers.{pod, provider, node, ...}`.

So import it from there.

It finds the table by the locator, but you can never ever use one table object against two different tables in UI. Even if they look similar. (because caching)